.storybook/main.js

```diff
@@ -19,4 +19,20 @@
 module.exports = {
 	stories: [ '../stories/**/*.stories.js', '../assets/js/**/*.stories.js' ],
 	addons: [ '@storybook/addon-viewport', '@storybook/addon-postcss' ],
+	previewHead: ( head ) => {
+		if ( ! process.env.VRT ) {
+			return head;
+		}
+
+		return `${ head }
+<style>
+	#root *,
+	#root *:before,
+	#root *:after {
```

Comment:
These tend to work, but technically these should be `::before` and `::after`, right? `:before` and `:after` are CSS2 🙂  https://developer.mozilla.org/en-US/docs/Web/CSS/::before

Also, do we want to cover things like `:hover`, just-in-case?

```suggestion
	#root *::before,
	#root *::after,
	#root *:hover,
	#root *:focus,
	#root *:active,
	#root *:visited {
```

Reply:
Thanks! I don't think adding `:hover` is necessary (I tested with this originally) as it should be covered by the base selector, especially since we're using `!important`.